src/components/container/BloodPressureVisualization/index.ts

```diff
@@ -0,0 +1 @@
+export {default} from "./BloodPressureVisualization";
```

Comment:
You'll also need to add exports to the container/index.ts for this to actually be exported from the library